.azure-pipelines.yml

```diff
@@ -0,0 +1,206 @@
+trigger: none # Do not build branches on push.
+pr:
+- master
+
+schedules:
+- cron: "0 0 * * *"
```

Comment:
Using midnight seems rather common, maybe use something else (like 3 or 4 in the morning) to increase the chance of not too many other build running?

Reply:
I think the classic argument against midnight builds is not valid in the cloud, because in the worst case the build gets queued. But I have changed it to 2, which is 1am/0pm in CET/CEST.